tensorflow/tensorboard/components/tf-dashboard-common/dashboard-style.html

```diff
@@ -54,6 +54,14 @@
         display: block;
       }
 
+      .log-option-button {
+          position: absolute;
+          margin-left: 26px;
```

Comment:
the log-option button did overlap the "download data" div. I moved the `.card-bottom-row` element over by 25px to get rid of the overlap.